### PR TITLE
JBDS-5001: Fix CRS coredump

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

**Review by jkopriva — APPROVED** [review]
LGTM and the fix is working, feel free to merge :)

File: site/com.jboss.devstudio.core.product [hunk 1]
```diff
@@ -23,8 +23,6 @@ Red Hat, Inc. - Initial implementation.
    <launcherArgs>
       <programArgs>-product
 com.jboss.devstudio.core.product
--showsplash
```
Comment (jeffmaury) [inline site/com.jboss.devstudio.core.product:26]: So will we get back to the upstream splash screen ? If yes then that's a no

Comment (sbouchet) [reply site/com.jboss.devstudio.core.product:26]: the splash path is configured in the config.ini file and correctly shown when starting eclipse. 
this is similar to https://bugs.eclipse.org/bugs/show_bug.cgi?id=575666.

Comment (jeffmaury) [reply site/com.jboss.devstudio.core.product:26]: Not sure to get it: which config.ini file ? Isn't it generated from this information ?

Comment (sbouchet) [reply site/com.jboss.devstudio.core.product:26]: https://user-images.githubusercontent.com/600414/131706249-5e47f0fd-3636-4b8e-ad0a-e10214b7ea77.mov

here is my local build without the splash in ini file

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**